sdk/python/cmd/pulumi-language-python/main.go

```diff
@@ -381,7 +381,7 @@ func (host *pythonLanguageHost) Pack(ctx context.Context, req *pulumirpc.PackReq
 		}
 	}
 
-	buildCmd, err := tc.ModuleCommand(ctx, "build", "--outdir", tmp)
+	buildCmd, err := tc.ModuleCommand(ctx, "build", "-w", "--outdir", tmp)
```

Comment:
Let's use the longform to make the flag more obvious.
```suggestion
	buildCmd, err := tc.ModuleCommand(ctx, "build", "--wheel", "--outdir", tmp)
```